std/traits.d

```diff
@@ -2714,7 +2713,8 @@ template Layout(T)
                         alias With = U;
                     else static if (U.length == 0)
                         alias With = T;
-                    else static if (T[0] <= U[0])
+                    else static if (T[0] < U[0]
+                            || T[0] == U[0] && T[1].stringof <= U[1].stringof)
```

Comment:
I'd highly discourage anyone from using stringof for anything but debugging.
It is certainly not for use in the stdlib.